src/Database/Connection.php

```diff
@@ -254,20 +325,26 @@ public function getDriver(): DriverInterface
      */
     public function connect(): bool
     {
-        try {
-            return $this->_driver->connect();
-        } catch (MissingConnectionException $e) {
-            throw $e;
-        } catch (Throwable $e) {
-            throw new MissingConnectionException(
-                [
-                    'driver' => App::shortName(get_class($this->_driver), 'Database/Driver'),
-                    'reason' => $e->getMessage(),
-                ],
-                null,
-                $e
-            );
+        foreach ($this->drivers as $role => $driver) {
```

Comment:
Seeing this I am happy we are dropping this method in 5.x :slightly_smiling_face:.